### PR TITLE
Added loadout 'Generic African Conventional Army'

### PR DESCRIPTION
**Pull request by Plankton555**
Added loadout for an african army.

Based largely on Soviet weapons. Some various uniforms are used, tried to find something that looked kinda similar to the Nigerian camo pattern. I've gone through all classes to make sure they "make sense" and don't overfill the inventory.

As for the integration into 1tac_misc, I haven't been able to test if that works (because of pbo packing and all that). I know the loadout in itself works as a mission-specific loadout. Likewise, if something must be added someplace else than CfgLoadouts, tell me.

### CODE REVIEW

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 1]
```diff
@@ -716,6 +716,12 @@ class CfgLoadouts
         category = "Other";
         #include "loadouts\afr_militia.hpp"
     };
+    class AFR_AR
```
Comment (beaar) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp:719]: AFR_ARMY or AFRICAN_ARMY would be more descriptive. I like to keep class and file names identical

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 2]
```diff
@@ -716,6 +716,12 @@ class CfgLoadouts
...
+    {
+        displayName = "Generic African Conventional Army (1990-)";
```
Comment (beaar) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp:721]: `African Army (1990-)` would sort better with the other african loadout

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp (hunk 1 above)
Comment (Plankton555) [reply addons/tm_tmf_loadouts/CfgLoadouts.hpp:719]: Huh, seems like I forgot half the word 😆 Will fix

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp (hunk 2 above)
Comment (Plankton555) [reply addons/tm_tmf_loadouts/CfgLoadouts.hpp:721]: Will fix

**Comment by beaar** [comment]
Thanks @Plankton555 😃

**Pull request merged**